engines/scumm/scumm-md5.h

```diff
@@ -1,5 +1,9 @@
 /*
+<<<<<<< HEAD
   This file was generated by the md5table tool on Mon Feb  7 12:42:06 2022
+=======
+  This file was generated by the md5table tool on Sat Feb 26 17:16:50 2022
+>>>>>>> 452db9ed48a (SCUMM: seperated AMIGA monkey VGA and VGA Demo variants. Assigned GF_DEMO flags)
```

Comment:
The merge conflict markers are still present. Please, resolve them

Reply:
done